routes/index.js

```diff
@@ -29,11 +30,29 @@ routes.POSTregister = function(req, res, next) {
     if (err) {
       console.log('error while user register!', err); return next(err);
     }
-    console.log('user registered!');
-
-    passport.authenticate('local')(req, res, function () {
-      res.send({redirect: '/'});
+    
+    //send email verification
+    //Used example: https://github.com/heitortsergent/passport-local-mongoose-email/tree/master/examples/login
+    var authenticationURL = 'http://localhost:3000/verify?authToken=' + user.authToken;
+    sendgrid.send({
+      to:       user.email,
+      from:     'SpringInitiative@olinjs.com',
+      subject:  'Confirm your email for Spring Initiative',
+      html:     '<a target=_blank href=\"' + authenticationURL + '\">Confirm your email</a>'
```

Comment:
Should add a "this is an automated response do not reply"
Also this might be misleading in the future since someone might try to email it for help, so eventually we can change it